bridge/github/config.go

```diff
@@ -207,7 +339,45 @@ func promptUsername() (string, error) {
 	}
 }
 
-func promptURL() (string, string, error) {
+func promptURL(remotes map[string]string) (string, string, error) {
+	validRemotes := valideGithubURLRemotes(remotes)
+	if len(validRemotes) > 0 {
+		for {
+			fmt.Println("\nDetected projects:")
+
+			// print valid remote github urls
+			for i, remote := range validRemotes {
+				fmt.Printf("[%d]: %v\n", i+1, remote)
```

Comment:
a nice touch would be to print the remote as the familiar github URL. That is, instead of printing `github.com:MichaelMure/git-bug.git`, having `github.com/MichaelMure/git-bug`

Reply:
Done. I changed the regex and used `strings.TrimSuffix` to remove the `.git` part